lib/hsa/kernels/sum_hfb.cl

```diff
@@ -0,0 +1,42 @@
+// Input data is float with sample-beam-freq, sum each frequency across samples
+//LWS = {64,  1  }
```

Comment:
It would be nice to have more comments here on what this kernel does, and what the global/local dimensions represent.